tests/h/search/query_test.py

```diff
@@ -97,6 +97,27 @@ def search(self, search):
 
 
 class TestGroupAuthFilter(object):
+    def test_does_not_return_annotations_if_group_not_readable_by_user(self,
+            search, Annotation, group_service):
```

Comment:
This triggers an [E128 warning](https://stackoverflow.com/questions/15435811/what-is-pep8s-e128-continuation-line-under-indented-for-visual-indent) in my editor's live linter. If we're adopting black's code style, it avoids this like so:

```
def test_does_not_return_annotations_if_group_not_readable_by_user(
    self, search, Annotation, group_service
):
```